DEVELOPMENT.md

```diff
@@ -0,0 +1,112 @@
+# Development
+This file highlight the installation process of getting started with Ersilia, Ersilia Artifacts and Creating a Pull Request.
+
+## Getting Started
+Ersilia can be run either on local computer or with the use of GitHub codespaces. Ersilia is designed to run on **Linux or Linux-like environments** namely: MacOS, Windows Sub-system for Linux (WSL) and at present is not supported on Windows systems.
+
+- ### Installation of Ersilia on local computer
+To get started with Ersilia installation, there are few dependencies that needs to be installed before proceeding to installation of Ersilia. Please, check the package requirement in the [Installation Guide](https://ersilia.gitbook.io/ersilia-book/ersilia-model-hub/installation)
+1. Once the required dependencies has been met, open a terminal and set up a Conda environment.
+
+```
+# create a conda environment
+conda create -n ersilia python=3.10
+# activate the environment
+conda activate ersilia
+```
+2. Clone the repository and run to install the Ersilia Python package.
+```
+# clone from github
+git clone https://github.com/ersilia-os/ersilia.git
+cd ersilia
+# install with pip (use -e for developer mode)
```

Comment:
Let's add a note on the developer mode. Something like "This will ensure that whenever you change the code, you do not need to reinstall ersilia for those changes to take effect".